modules/serving-transport-encryption-configuring-selfsigned-clusterissuer.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: PROCEDURE
```

Comment:
Same as previous comments, make sure that proper metadata for modules is included